doc/user-guide.md

```diff
@@ -0,0 +1 @@
+# Complete User guide.
```

Comment:
Maybe a link to the wiki would be nice here. Also on the README.md

Reply:
Done!!